documentation/src/test/java/example/exception/AssertDoesNotThrowExceptionDemoTestCase.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2015-2024 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package example.exception;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+// @formatter:off
+// tag::user_guide[]
+
+class AssertDoesNotThrowExceptionDemoTestCase {
+
+	@Test
+	void assertDoesNotThrowExceptionTest() {
+		Assertions.assertThrows(RuntimeException.class, () -> {
+			failsDueToException();
+			Assertions.assertDoesNotThrow(() -> {
+				shouldNotThrowException();
+			});
```

Comment:
Please move this to a separate example.

Reply:
I created separate sections for `assertThrows`, `assertThrowsExactly`, and `assertDoesNotThrow` and moved individual examples accordingly.